codeforboston/maple
# Follow btn individual users

# Summary
Adds the follow button and request to individual users both on their profile and on each search result card. #1592

# Checklist


# Screenshots

<img width="1385" alt="Screenshot 2024-12-03 at 8 31 33 PM" src="https://github.com/user-attachments/assets/5d6fb272-d3a9-466f-bcb4-d46dcd6c833b">
<img width="1026" alt="Screenshot 2024-12-03 at 8 31 01 PM" src="https://github.com/user-attachments/assets/779a888c-a6cb-43e3-b864-14f6c423381b">
<img width="1005" alt="Screenshot 2024-12-03 at 8 30 43 PM" src="https://github.com/user-attachments/assets/9843b2b7-f6b2-4cd5-a5a0-3607829b8780">

# Steps to test/reproduce

1. Go to a user's profile page
2. Notice the Follow button
3. Click on the button
4. Go to Browse Testimony page, notice Follow button on cards that are ind. users
5. Click on the button
6. Your own profile should not have this button
